acceptance-tests/package.json

```diff
@@ -17,10 +17,10 @@
     "url": "git+https://github.com/HubSpot/hubspot-cli.git"
   },
   "scripts": {
-    "test": "vitest run",
-    "test-debug": "DEBUG=true vitest run",
-    "test-ci": "CI=true vitest run",
-    "test-qa": "QA=true vitest run"
+    "test": "yarn && vitest run",
+    "test-debug": "DEBUG=true yarn && vitest run",
+    "test-ci": "CI=true yarn && vitest run",
+    "test-qa": "QA=true yarn && vitest run"
```

Comment:
I'm on the fence about prefixing all of these with `yarn` instead of running the install once before running these commands.  

Also I think the commands with env variables might be incorrect.  I think they would need to `yarn && ENV_VARIABLE second command` so the env variable applies to the second command.

Reply:
Ah good to know, I didn't realize env variables only applied to the immediate following command. But yeah that makes sense. So just add a `yarn` to the gh workflow and then do it manually when you need to run locally?

Reply:
Yeah, that sounds good!